main/app-menu.js

```diff
@@ -0,0 +1,56 @@
+const { Menu, MenuItem, ipcMain } = require('electron')
+const { ipcMainEvents } = require('./ipc')
+
+function setupAppMenu (/** @type {import('./typings').Context} */ ctx) {
+  if (process.platform !== 'darwin') return
+
+  // Add "Check for updates..." item to the Application menu on MacOS
```

Comment:
Did you consider Windows for this PR as well? If we don't add it now then an issue will help keeping the implementations in sync in the future

Reply:
We are thinking alike 😀

https://github.com/filecoin-project/filecoin-station/pull/62#discussion_r931039942